src/components/admin/MainIssueController.jsx

```diff
@@ -156,8 +156,26 @@ export default class MainIssueController extends FalcorController {
               return false;
             }
           }
+          if (/<a.*?href\s*?=\s*?["'](?!http).*?>/.test(article.html)) {
+            const url = article.html.match(/<a.*?href\s*?=\s*?["'](?!http)(.*?)["'] *?>/)[1];
+            if (!window.confirm(
+              `${article.title} with URL ${url} is in non-absolute format, ` +
+              'which means that it does not have http(s):// in front of it, ' +
+              'which will break the link. ' +
+              'It will be misinterpreted and it will simply add the ' +
+              'link written to the end of the URL. ' +
+              'The easiest way to get a correct link is to copy it ' +
+              'from your browsers URL bar, remember to prefer https over http. ' +
```

Comment:
browser's* (we work for a newspaper, they'll roast us if we screw up grammar :P)